operations/cancellationpolicies.md

```diff
@@ -91,22 +94,29 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `RateGroupId` | string | required | Unique identifier of the rate group the cancellation policy belongs to. |
 | `CreatedUtc` | string | required | Date and time of the cancellation policy creation in UTC timezone in ISO 8601 format. |
 | `UpdatedUtc` | string | required | Date and time of the cancellation policy update in UTC timezone in ISO 8601 format. |
-| `Applicability` | string [Applicability](#cancellation-policy-applicability) | required | Applicability mode of the cancellation policy. |
-| `FeeExtent` | array of string [Fee Extent](#fee-extent) | required | Extent for the cancellation fee, i.e. what should be in scope for the automatic payment. |
-| `ApplicabilityOffset` | string | required | Offset for order start (assuming Applicability is set to `Start`) from which the fee is applied. |
-| `FeeMaximumTimeUnits` | int | required | Maximum number of time units the cancellation fee is applicable to. |
-| `AbsoluteFee` | [Currency value](../operations/accountingitems.md#currency-value) | optional | Absolute value of the fee. |
-| `RelativeFee` | decimal | required | Relative value of the fee, as a percentage of the reservation price. |
+| `Applicability` | [Cancellation Policy Applicability](cancellationpolicies.md#cancellation-policy-applicability) | required | Applicability mode of the cancellation policy. |
+| `FeeExtent` | array of [Cancellation Fee Extent](cancellationpolicies.md#cancellation-fee-extent) | required | Extent for the cancellation fee, i.e. what should be in scope for the automatic payment. |
+| `ApplicabilityOffset` | string | required | Offset for order start (assuming Applicability is set to Start) from which the fee is applied. |
+| `FeeMaximumTimeUnits` | integer | required | Maximum number of time units the cancellation fee is applicable to. |
+| `AbsoluteFee` | [Currency value (ver 2023-02-02)](cancellationpolicies.md#currency-value-ver-2023-02-02) | required | Absolute value of the fee. |
+| `RelativeFee` | number | required | Relative value of the fee, as a percentage of the reservation price. |
+| `IsActive` | boolean | required | Whether the cancellation policy is still active. |
 
 #### Cancellation Policy Applicability
 
-* `Creation` - Cancellation fee is applicable from the time of creating the reservation.
-* `Start` - Cancellation fee is applicable as soon as the reservation starts, i.e. at arrival time.
-* `StartDate` - Cancellation fee is applicable on the date the reservation starts, i.e., at midnight.
-* ...
+* `Creation`
+* `Start`
+* `StartDate`
 
-#### Fee Extent
+#### Cancellation Fee Extent
 
 * `TimeUnits`
 * `Products`
-* ...
+
+#### Currency value (ver 2023-02-02)
+Absolute value of the fee.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Currency` | string | required |  |
+| `Value` | number | required |  |
```

Comment:
issue: This one should go under [common objects page](https://mews-systems.gitbook.io/connector-api/operations/_objects). So please:

1. Past this parts to the `_objects.md` file
2. In `types.yaml`, change `file` under `currencyvalue` from `cancellationpolicies.md` to `_objects.md`
3. Rerun the generator

The generator ignores `_objects.md` file (it's manually maintained), but it will create correct links to it and removes this part from this file.

Reply:
sss